java/code/src/com/suse/manager/webui/controllers/channels/ChannelsApiController.java

```diff
@@ -77,6 +79,12 @@ public static String getAllChannels(Request req, Response res, User user) {
                             .filter(c -> !filterOutIds.contains(c.getBase().getId()))
                             .collect(Collectors.toList()
             );
+            Collections.sort(jsonChannelsFiltered, new Comparator<ChannelsJson>() {
+                @Override
+                public int compare(ChannelsJson c1, ChannelsJson c2) {
+                    return c1.getBase().getId().compareTo(c2.getBase().getId());
+                }
+            });
```

Comment:
It works just fine, but here's a simpler one-liner:
```suggestion
            jsonChannelsFiltered.sort(Comparator.comparing(cIn -> cIn.getBase().getId()));
```

Reply:
Nice, thanks a lot for this. I don't know enough about Java so I wouldn't even have thought of something like this. 🍻